openghg/retrieve/_access.py

```diff
@@ -738,13 +739,13 @@ def _metadata_difference(
     for param in param_difference:
         summary_difference[param] = []
         if print_output:
-            print(f" {param}: ", end="")
+            logging.debug(f" {param}: ")
```

Comment:
Does debug get returned as a printed output? Just thinking for an explicit `print_output` input it would be useful to ensure this does actually print something out for the user.

Reply:
Ah yeah we do set the level to be `logging.DEBUG` so maybe this is ok. Would be good to check.

Reply:
I've set `consoleHandler.setLevel(logging.INFO)` down from `consoleHandler.setLevel(logging.WARNING)` and changed this to `logger.info` now.